src/log-store/Cargo.toml

```diff
@@ -40,6 +40,7 @@ tokio.workspace = true
 [dev-dependencies]
 common-meta = { workspace = true, features = ["testing"] }
 common-test-util.workspace = true
+common-wal = { workspace = true, features = ["testing"] }
```

Comment:
Doesn't seem we need this dependency?

Reply:
When execute under `log-store`, like ` cargo test --package log-store --lib -- raft_engine::backend::tests::test_compare_and_put --exact --nocapture`, error occurred:
<img width="974" alt="image" src="https://github.com/GreptimeTeam/greptimedb/assets/40162016/32883cb7-dc57-427a-83fd-8d32dec4c594">

`common_wal::test_util` needs testing feature: https://github.com/realtaobo/greptimedb/blob/a50025269f5fc794e1959b282761cfa45a8e2e2d/src/common/wal/src/lib.rs#L27